.github/wasi_testsuite_skip.json

```diff
@@ -1,5 +1,4 @@
 {
   "WASI Rust tests": {
-    "interesting_paths": "config is broken, and after fixing it forbids leading slash with EPERM"
```

Comment:
this fails because the test was merged without configuration to run it. locally, it passes now, though

Reply:
https://github.com/WebAssembly/wasi-testsuite/pull/67#issuecomment-1478014817 what's needed